bucket/amazon-workspaces.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.1.4",
+    "description": "Client for Amazon workspaces service",
+    "homepage": "https://clients.amazonworkspaces.com",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://clients.amazonworkspaces.com/app-terms.html"
+    },
+    "url": "https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi",
+    "hash": "604e40ed03883e1d740367fbcc4da9c21fc88fe26f9df7aca4ed0a25c0b1ac2e",
+    "extract_dir": "[ApplicationFolderName]",
+    "shortcuts": [
+        [
+            "workspaces.exe",
+            "Amazon Workspaces"
+        ]
+    ],
+    "checkver": {
+        "url": "https://d2td7dqidlhjx7.cloudfront.net/prod/dub/windows/WorkSpacesAppCast.xml",
+        "regex": "title>Version ([\\d\\.]+)</"
+    },
+    "autoupdate": {
+        "url": "https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi"
+    },
+    "post_install": [
+        "#disable the autoupdate of amazon-workspaces client.",
+        "$configFile = \"$dir\\workspaces.dll.config\" ",
+        "If (Test-Path -Path $configFile -ErrorAction SilentlyContinue) { ",
+        "   [xml]$xml = Get-Content -path $configFile ",
+        "   $key = $xml.SelectNodes('//configuration/appSettings/*') | Where-Object {$_.key -eq 'UpdateUrl'}",
```

Comment:
Does not make sense to use where-object with combination of xpath selection

Reply:
I added attribute filter in the xpath query and switched to SelectSingleNode method.